internal/services/notification/db/db.go

```diff
@@ -306,3 +303,35 @@ func (d *DB) DeleteCommitStatusDeliveriesByCommitStatusID(tx *sql.Tx, commitStat
 
 	return nil
 }
+
+func (d *DB) GetProjectCommitStatusDeliveriesAfterSequenceByProjectID(tx *sql.Tx, afterSequence uint64, projectID string, deliveryStatusFilter []types.DeliveryStatus, limit int, sortDirection types.SortDirection) ([]*types.CommitStatusDelivery, error) {
```

Comment:
Wrong position for Get function, should be with the other get function. Also `DeleteRunWebhookDeliveriesByRunWebhookID` is in the wrong position between commit status related functions. Please reorder.